checker-plugin/src/main/scala/org/virtuslab/akkasaferserializer/AkkaSerializabilityCheckerPluginComponent.scala

```diff
@@ -13,6 +13,8 @@ class AkkaSerializabilityCheckerPluginComponent(val pluginOptions: PluginOptions
 
   override def newPhase(prev: Phase): Phase =
     new StdPhase(prev) {
+      private val serializabilityTraitType = typeOf[SerializabilityTrait]
```

Comment:
Hmmm so what was the exact cause of `NoClassDefFoundError`? 🤔

Reply:
https://github.com/sbt/sbt/issues/2255
CheckerPlugin is quite basic, so this solution works.
DumpSchemaPlugin will require one of the workarounds suggested in this issue.